docs/using-the-compiler.rst

```diff
@@ -287,8 +287,14 @@ Input Description
               // Improve allocation of stack slots for variables, can free up stack slots early.
               // Activated by default if the Yul optimizer is activated.
               "stackAllocation": true,
-              // Select optimization steps to be applied.
-              // Optional, the optimizer will use the default sequence if omitted.
+              // Select optimization steps to be applied. It is also possible to modify both the
+              // optimization sequence and the cleanup sequence. Instructions for each sequence
+              // are separated with the : delimiter. For more information see
+              // "The Optimizer > Selecting Optimizations".
+              // Optional, the optimizer will use the default sequence if omitted. Omitting the
+              // whole field will make the compiler use defaults for both sequences but setting
+              // it to an empty value (":") will not - it will instead make the compiler use
+              // only the default cleanup sequence.
```

Comment:
```suggestion
              // Select optimization steps to be applied. This field is optional, and if omitted, the 
              // default optimization sequence will be applied.
              // The optimizer sequence contains both the optimization steps as well as the clean-up
              // sequence steps, delimited by a colon ( : ). Omitting either value will force the compiler 
              // to not apply the omitted sequence.
              // For more detailed information, refer to:
              // https://docs.soliditylang.org/en/latest/internals/optimizer.html#selecting-optimizations
```

Reply:
As I said in https://github.com/ethereum/solidity/pull/13619#discussion_r991282501, I think it's also important to explicitly mention that simply setting this to an empty string does not clear both sequences. It's implied but this is the most important use case so we should make it really clear.